x-pack/platform/plugins/shared/alerting/server/application/backfill/methods/delete/delete_backfill.ts

```diff
@@ -82,12 +84,40 @@ async function deleteWithOCC(context: RulesClientContext, { id }: { id: string }
       })
     );
 
+    const actionsClient = await context.getActionsClient();
+
+    const backfillResult = transformAdHocRunToBackfillResult({
```

Comment:
would it be easier just to use the `getBackfill` method instead of repeating the transformation here?